cfme/tests/intelligence/reports/test_crud.py

```diff
@@ -264,3 +263,26 @@ def test_reports_delete_saved_report(appliance, request):
     view.configuration.item_select(
         item='Delete selected Saved Reports', handle_alert=True)
     assert not report.exists
+
+
+def test_reports_crud_schedule_for_base_report_once(appliance, request):
```

Comment:
I think this case is covered by test_schedule_queue?  

https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/intelligence/reports/test_reports_schedules.py

This uses several schedules from the cfme/data/ directory.